packages/user-management-widget/src/lib/widget/drivers/BaseDriver.ts

```diff
@@ -3,6 +3,21 @@ import { Logger } from '../../mixins/loggerMixin/types';
 type Empty = null | undefined;
 type RefOrRefFn = Element | (() => HTMLElement | Empty) | Empty;
 
+const waitForElement = async (ele: RefOrRefFn, timeout: number) =>
```

Comment:
Helper file?

Reply:
Done. Also moved types to a separate file to avoid circular dependency.